### ISSUE TITLE
Permission denied on "Interleave the Sequences"

### ISSUE THREAD

**Issue by smack33** [dib-lab/eel-pond#39]
Data was originally in a volume attached to a separate instance. When detached and attached to a new instance, the permissions changed to read-only. This did not cause any issues until the interleave step. We solved by `sudo chmod a+rwxt *.gz` Screenshot of error attached.
![interleave error](https://cloud.githubusercontent.com/assets/25188559/22046062/194e84f8-dcd3-11e6-9c92-bc13ff4f05bb.png)

**Comment by johnsolk** [comment]
This is on Jetstream, btw.

**Comment by standage** [comment]
From slack it seems like this isn't holding you up at all, @smack33. Should we leave this open as a note to improve the documentation, or was this more of an FYI?

**Comment by johnsolk** [comment]
I think note to improve documentation. Will there be a Jetstream set of instructions? @standage

**Comment by standage** [comment]
Yep, on my list of things I'm <strike>ignoring</strike> working on.
